package-lock.json

```diff
@@ -0,0 +1,5901 @@
+{
```

Comment:
This seems like a good thing to add to .gitignore

Reply:
It's noisy but they recommend committing it